src/services/legal-services.ts

```diff
@@ -252,18 +252,62 @@ export default class LegalServices {
     })
   }
 
-  /** Business Digital Credential Helpers **/
+  /**
+   * Gets a pre-signed URL for the specified filename.
+   * @param filename the file name
+   * @returns a document upload object
+   */
+  static async getPresignedUrl (fileName: string): Promise<DocumentUploadIF> {
+    const url = `documents/${fileName}/signatures`
+    return axios.get(url)
+      .then(response => {
+        const data = response?.data
+        if (!data) throw new Error('Invalid API response')
+        return data
+      }).catch(error => {
+        // eslint-disable-next-line no-console
+        console.log('getPresignedUrl() =', error)
+        throw error
+      })
+  }
+
+  /**
+   * Uploads the specified file to the specified URL.
+   * @param url the URL to upload to
+   * @param file the file to upload
+   * @param key the file key
+   * @param userId the file user id
+   * @returns the axios response
+   */
+  // *** TODO: find out type of error response
+  static async uploadToUrl (url: string, file: File, key: string, userId: string): Promise<AxiosResponse> {
+    const headers = {
+      'Content-Type': file.type,
+      'x-amz-meta-userid': `${userId}`,
+      'x-amz-meta-key': `${key}`,
+      'Content-Disposition': `attachment; filename=${file.name}`
+    }
+    return axios.put(url, file, { headers })
+      .then(response => response)
+      .catch(error => error.response)
+  }
+
+  //
+  // Business Digital Credential Helpers
+  //
 
   /**
    * Fetches digital credential information.
    * @param businessId the business identifier (aka entity inc no)
    * @returns a promise to return the credentials connection data
    */
-  static async fetchCredentials (businessId: string): Promise<any> {
+  static async fetchCredentials (businessId: string): Promise<AxiosResponse> {
     const url = `businesses/${businessId}/digitalCredentials`
     return axios.get(url)
       .catch(error => {
+        // eslint-disable-next-line no-console
         console.log(error.message)
+        return null
```

Comment:
Without this, there was a warning about an undefined return value (which was OK with return type `any` but not OK with return type `AxiosResponse`).